app/assets/stylesheets/cards.scss

```diff
@@ -10,7 +10,7 @@
   border: none;
 
   a.btn {
-    color: $link-light-color;
+    --bs-body-color: $link-light-color;
```

Comment:
I am wondering if the conversion to css was not finishing for this particular file, both for this variable and for the file extension.

Reply:
We can't convert this file to css because it uses `@include`.